### DISCUSSION TITLE
CHARGING_HISTORY now uses bff/v2/mobile-app/charging/history

### DISCUSSION THREAD

**Discussion by mstoyanovv** [tdorssers/TeslaPy#61] [Q&A] [answered]
I am trying to retrieve the data that 'Charge Stats' screen is using but so far without luck. 
Looking at the latest ownerapi-endpoint the request for charging_history has changed to bff/v2 endpoint. 
Anyone that what a request to this endpoint would look like?

**Comment by tdorssers** [comment]
Does your "access_type" equals "DRIVER" or something else?

**Comment by mstoyanovv** [comment, marked as answer]
My issues was that Tesla expects this header - 'x-tesla-user-agent': 'TeslaApp/4.7.0' when making a request to /charge_history. Without it it returns "{}/n". Adding it should fix the empty charging data screen in 'gui.py'

**Reply by tdorssers** [reply]
Thanks! It also made recent endpoints working, like REMOTE_AUTO_SEAT_CLIMATE_REQUEST.

**Reply by FashionCStar** [reply]
Hi, @tdorssers & @mstoyanovv 
Hope you are doing well.
I am also going to use charge_history api endpoint, but not sure what request parameters should I send when I call api.

[https://tesla-api.timdorr.com/miscellaneous/endpoints](https://tesla-api.timdorr.com/miscellaneous/endpoints)
  "VEHICLE_CHARGE_HISTORY": {
    "TYPE": "GET",
    "URI": "api/1/vehicles/{vehicle_id}/charge_history",
    "AUTH": true
  },

What data should pass on the request?
